### PR TITLE
Estimate delivery time

### PR DESCRIPTION
**Pull request by yrong**
More context in https://snowfork.slack.com/archives/G01BGMGLAC9/p1720027597396119

Requires: https://github.com/Snowfork/snowbridge-subsquid/pull/1

### CODE REVIEW

File: web/packages/api/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "@snowbridge/contract-types": "workspace:*",
     "@typechain/ethers-v6": "^0.5.1",
     "ethers": "^6.12.1",
-    "rxjs": "^7.8.1"
+    "rxjs": "^7.8.1",
+    "axios": "^1.7.2"
```
Comment (alistair-singh) [inline web/packages/api/package.json:39]: Lets remove dependencies on axios and use standard `fetch` api.

File: web/packages/api/src/utils.ts [hunk 2]
```diff
@@ -117,3 +118,10 @@ export const fetchFinalityUpdate = async (
         attested_slot: Number(result?.data?.attested_header?.beacon?.slot),
     }
 }
+
+export const fetchEstimatedDeliveryTime = async (graphqlUrl: string, channelId: string) => {
+    let response = await axios.post(graphqlUrl, {
```
Comment (alistair-singh) [inline web/packages/api/src/utils.ts:123]: Lets just use fetch here.

**Review by alistair-singh — CHANGES_REQUESTED** [review]
Minor comment about removing axios dependency.

File: web/packages/api/src/utils.ts (hunk 2 above)
Comment (yrong) [reply web/packages/api/src/utils.ts]: https://github.com/Snowfork/snowbridge/pull/1256/commits/40030b447cf4fe970b2fb43d343936803cbad584

**Review by alistair-singh — APPROVED** [review]

**Pull request merged**